providers/fips/self_test_data.inc

```diff
@@ -422,3 +496,687 @@ static const ST_KAT_DRBG st_kat_drbg_tests[] =
         ITM(drbg_hmac_sha1_pr_expected)
     }
 };
+
+/* KEY EXCHANGE TEST DATA */
+
+#ifndef OPENSSL_NO_DH
+/* DH KAT */
+static const unsigned char dh_p_hex[] = {
```

Comment:
Why _hex suffix? This applies to other data and algos below as well.

Reply:
removed